src/extract/transpile/vue-template-wrap.js

```diff
@@ -11,48 +11,39 @@ const { supportedTranspilers } = require("../../../src/meta.js");
  * version of vue-template-compiler) if the first one fails
  */
 function getVueTemplateCompiler() {
-  let lIsVue3 = false;
-
-  let lCompiler = tryRequire(
-    "vue-template-compiler",
-    supportedTranspilers["vue-template-compiler"]
-  );
-
-  if (lCompiler === false) {
-    lCompiler = tryRequire(
-      "@vue/compiler-sfc",
-      supportedTranspilers["@vue/compiler-sfc"]
-    );
-    lIsVue3 = true;
-  }
-
-  return { lCompiler, lIsVue3 };
+	let lIsVue3 = false;
+
+	let lCompiler = tryRequire("vue-template-compiler", supportedTranspilers["vue-template-compiler"]);
+
+	if (lCompiler === false) {
+		lCompiler = tryRequire("@vue/compiler-sfc", supportedTranspilers["@vue/compiler-sfc"]);
+		lIsVue3 = true;
+	}
+
+	return { lCompiler, lIsVue3 };
 }
 
-const { lCompiler: vueTemplateCompiler, lIsVue3: isVue3 } =
-  getVueTemplateCompiler();
+const { lCompiler: vueTemplateCompiler, lIsVue3: isVue3 } = getVueTemplateCompiler();
 
 function vue3Transpile(pSource) {
-  const parsedComponent = vueTemplateCompiler.parse(pSource);
-  const errors = _get(parsedComponent, "errors");
+	const parsedComponent = vueTemplateCompiler.parse(pSource);
+	const errors = _get(parsedComponent, "errors");
+
+	if (!isEmpty(errors)) {
+		return "";
+	}
 
-  if (!isEmpty(errors)) {
-    return "";
-  }
+	const scriptContent = _get(parsedComponent, "descriptor.script.content", "");
+	const scriptSetupContent = _get(parsedComponent, "descriptor.scriptSetup.content", "");
 
-  return _get(parsedComponent, "descriptor.script.content", "");
+	return scriptContent + scriptSetupContent;
```

Comment:
Let's say scriptContent is `const lThing = 123` and scriptSetupContent `import {hello} from 'some-module'` (both syntactically valid) - the concatenated code would look like this (which isn't valid):

```javascript
const lThing = 123import {hello} from 'some-module'
```

... unless the template compiler ensures there's a line break at the end of each of them. 

=> Would inserting a line break (os.EOL) mitigate this problem sufficiently?

Reply:
done